packages/components/src/components/DiagramFlamegraph.vue

```diff
@@ -42,10 +41,6 @@ export default {
       type: Array,
       required: true,
     },
-    selectedEvents: {
-      type: Array,
```

Comment:
The diagram is already being passed an array of events to select / highlight / focus (the terminology used in the diagram is very confusing - I couldn’t quite understand it). We may not need to create a new highlight mechanism here. We can just use the list that’s already passed in. @dustinbyrne

Reply:
The selected object is what is displayed in the sidebar, and it gets centered and highlighted in some way in the diagram by default. 

The focused object is not displayed in the sidebar, but it gets centered and highlighted. The little eyeball icon uses this.

Both of these highlight a single event/code object, whereas I'm trying to highlight multiple events in a distinct way with the new highlight mechanism.

Reply:
See my other comment regarding prop drilling